### PR TITLE
Fix CI dependecies and flow

### PR DESCRIPTION
**Pull request by yanirq**

### CODE REVIEW

**Comment by yanirq** [comment]
@gbenhaim  @danielBelenky  can you guys say why the CI is failing on python3 ? i tired several options here

**Comment by davidvossel** [comment]
> @gbenhaim @danielBelenky can you guys say why the CI is failing on python3 ? i tired several options here

does automation/test.sh need to install python3?

**Comment by yanirq** [comment]
> > @gbenhaim @danielBelenky can you guys say why the CI is failing on python3 ? i tired several options here
> 
> does automation/test.sh need to install python3?

dnf/yum commands inside the script cannot be found when attempted to add:
yum install -y python3 for example

File: automation/check-patch.k8s-1.11.0.packages [hunk 1]
```diff
@@ -1,2 +1,3 @@
 docker
 make
+python3
```
Comment (gbenhaim) [inline automation/check-patch.k8s-1.11.0.packages]: should be `python36` (also in the other *package files)

**Pull request merged**